datagateway_api/src/datagateway_api/database/models.py

```diff
@@ -1151,6 +1545,23 @@ class SHIFT(Base, EntityHelper, metaclass=EntityMeta):
     )
 
 
+class TECHNIQUE(Base, EntityHelper, metaclass=EntityMeta):
```

Comment:
Can you test if the related entity from technique to datasettechnique works please? There's no relationship in this class currently - it might work due to the relationship in technique (cause back referencing) but it would be good to check. Again, like my other testing review comment, it might be easier to work on the generator script and come back to this in the future.